### PR TITLE
Eli/swap admin start

### PR DESCRIPTION
**Pull request by EGuenz**
Admin checks start entrypoint of swap contracts and updates tests

### CODE REVIEW

File: packages/minter-contracts/ligo/src/swaps/fa2_allowlisted_swap_offchain_nocounter.mligo [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+let rec offchain_accept_batch_helper(ps, (ops, storage) : permit_accept_param list * (operation list * storage)) 
+  : allowlist_return = 
+    match ps with 
+      | [] -> (ops, storage)
+      | permit :: remaining_permits -> 
+          let (ops, storage) = offchain_accept(permit, storage, ops) in
+          offchain_accept_batch_helper(remaining_permits, (ops, storage))
+
+let offchain_accept_batch(ps, storage : permit_accept_param list * storage)
```
Comment (michaeljklein) [inline packages/minter-contracts/ligo/src/swaps/fa2_allowlisted_swap_offchain_nocounter.mligo:35]: can this be replaced with a fold?

**Review by michaeljklein — APPROVED** [review]
LGTM: small comment on loop vs. fold as on #201

File: packages/minter-contracts/ligo/src/swaps/fa2_allowlisted_swap_offchain_nocounter.mligo (hunk 1 above)
Comment (EGuenz) [reply packages/minter-contracts/ligo/src/swaps/fa2_allowlisted_swap_offchain_nocounter.mligo:36]: Fixed thanks

**Review by michaeljklein — APPROVED** [review]
LGTM

**Pull request merged**